nginx_stage/lib/nginx_stage/configuration.rb

```diff
@@ -72,7 +72,7 @@ module Configuration
     attr_writer :pun_custom_env
 
     def pun_custom_env
-      transform_keys(@pun_custom_env) { |key| key.to_s }
+      Hash[@pun_custom_env.map {|key, value| [key.to_s, value.to_s]}]
```

Comment:
This would be slightly more readable if the helper method was:

    transform_keys_and_values_to_strings(@pun_custom_env)

and the separate method

```ruby
def transform_keys_and_values_to_strings(obj)
  Hash[obj.map {|key, value| [key.to_s, value.to_s]}]
end
```

Or a comment? Of course we have more complex transformations than this that don't have comments that I've written in the past so...

Reply:
I elected to remove the other method because it was only used in this one method. It seems unnecessary to shift the main implementation of `pun_custom_env` to another method if there is no present need to make that section of code re-usable. I am open to the argument that a method to ensure that a Hashes keys and values are strings is useful, but then I question: is `NginxStage::Configuration` the correct place for such a thing?